src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -92,6 +91,12 @@ public boolean handle(int code) {
             });
         }
     }
+    
+    private void setupSecurity(Settings settings, Environment environment) throws Exception {
+        if (settings.getAsBoolean("security.manager.enabled", true)) {
```

Comment:
It is a constant. Just like all the other strings in this file :)

So I will add it, but I will do so under protest:
* there are zero named constants in this file, I am just matching the style surrounding the code, for consistency.
* named constants only used once just cause indirection hurt locality of reference to those reading the code.

In general, named constants considered harmful.
*